templates/admins.ini.j2

```diff
@@ -7,7 +7,7 @@
 {% if couchdb_admins is defined %}
 [admins]
 {% for (userid, pwd) in couchdb_admins.iteritems() %}
-{{userid}} = '{{pwd|e}}'
+{{userid}} = '{{pwd|couchdb_password_hash(userid|hash('sha1'))}}'
```

Comment:
this sets the password hash instead of the plain text password so that if you role is re-run there will be no diffs